internal/version/tool_latest_version_finder.go

```diff
@@ -21,16 +21,20 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
-	"sort"
 	"strings"
 	"time"
 
 	"github.com/Masterminds/semver/v3"
 	"github.com/google/go-github/v38/github"
 )
 
+const (
+	minPageSize = 5
+)
+
 type LatestVersionFinder interface {
 	PrintNewVersionAvailable(w io.Writer, v, tool, bin string) error
+	IsValidTagForTool(tag, tool string) bool
```

Comment:
[q] do we need to export this method? can we keep it private?

Reply:
> I am exporting for testing purposes I think it's important to keep it covered

if unit testing package private are still testable, given this is an interface the most important question, are consumers of `LatestVersionFinder` need to use `IsValidTagForTool`?